app/assets/stylesheets/components/_modal.scss

```diff
@@ -61,28 +62,33 @@
 .modal {
   &-sm {
     max-height: 50%;
+    max-width: 50%;
     width: 45%;
   }
 
   &-md {
     max-height: 70%;
+    max-height: 65%;
```

Comment:
what is the reason to maintain two `max-height`?

Reply:
my bad, have to be width, skill issue :(